cypress/fixtures/example.json

```diff
@@ -0,0 +1,5 @@
+{
+  "name": "Using fixtures to represent data",
+  "email": "hello@cypress.io",
+  "body": "Fixtures are a great way to mock data for responses to routes"
+}
```

Comment:
If this file is not required, we should remove it.
But as this directory should actually exist, we can have a [.keep](https://stackoverflow.com/questions/29183372/rails-why-is-there-keep-file-in-every-directory) file here.